src/main/java/usecase/TTCUseCase.java

```diff
@@ -1,7 +1,12 @@
 package usecase;
 
+import entity.GameData;
+import entity.Player;
+import usecase.impactor.PositionImpactor;
+
 /**
  * Represent the use case when a player arrives at the TTC station block
  */
 public class TTCUseCase {
+
 }
```

Comment:
there is nothing, should we delete this file?

Reply:
yes, I think It's better to delete